static/app/views/issueList/header.tsx

```diff
@@ -228,10 +229,6 @@ function IssueListHeader({
 
 export default IssueListHeader;
 
-const StyledLayoutTitle = styled(Layout.Title)`
```

Comment:
`Layout.Title` will probably get replaced by `StyledPageHeading` in the future, right now we have a lot more `PageHeading` so i'll just standardize on that and switch it all over to a new `Layout.Title` later.